frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionRegisteredDeploymentsView.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { ModelVersion } from '~/concepts/modelRegistry/types';
+import useModelArtifactsByVersionId from '~/concepts/modelRegistry/apiHooks/useModelArtifactsByVersionId';
+import EmptyModelRegistryState from '~/pages/modelRegistry/screens/EmptyModelRegistryState';
+
+type ModelVersionRegisteredDeploymentsViewProps = {
+  modelVersion: ModelVersion;
+};
+
+const ModelVersionRegisteredDeploymentsView: React.FC<
+  ModelVersionRegisteredDeploymentsViewProps
+> = ({ modelVersion: mv }) => {
+  const [modelArtifact] = useModelArtifactsByVersionId(mv.id);
```

Comment:
Don't worry about the data here, we won't use model artifacts, but rather InferenceServices objects, we'll need to create other context and hooks for that, leave a TODO there and we can call it a day for this PR